glos_qartod/qc.py

```diff
@@ -235,13 +297,21 @@ def apply_qc(self, ncvariable):
             test_params['arr'] = values
 
         if values.size > 0:
-           qc_flags = qc_tests[qartod_test](**test_params)
+            # Try to run the test.  If it fails, return an exception
+            try:
+                qc_flags = qc_tests[qartod_test](**test_params)
+            except Exception as e:
+                get_logger().error("QARTOD test application failed: %s", str(e))
```

Comment:
Please use `get_logger().exception` here instead. That way you don't need to include the str(e) and it will print a stack trace.